images/kube-test/runner

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+# hostpath for docker container launched in verify container; cleanup hostpath
+pushd .
+cd /go/src/k8s.io/kubernetes
+rm -rf .[^.] .??*
+rm -rf *
+popd
+
+# symbol link doesn't work for verify
+mkdir -p /go/src/github.com/$REPO_OWNER/kubernetes
```

Comment:
I don't understand what this is doing or why we need this.

Reply:
This is only need when this job runs against a fork repository of kubernetes where bootstrap put the code into github.com/<fork_org>/kubernetes, instead of k8s.io/kubernetes, as we deployed our own prow envrionment